Adds Sketch document with examples.

ryanmiglavs opened the pull request:
I've added a Sketch design doc, with examples of a closed menu and then open with three title positions.

Let me know if you'd like this organized differently.

Also, if this gains use, I'm happy to make a dark theme as well.

TwoLivesLeft commented:
Looks great! Though I noticed an issue when loading in my copy of Sketch

https://twitter.com/twolivesleft/status/1082432592345915392

Perhaps we should drop the trailing ellipsis from the "Insert Image" and "Insert Link" items? Or somehow resolve the wrapping there.

Do you think you could also change the folder name to just "Design"? 

I really like the neutral colours and default iOS blue. I will probably update the default light theme to reflect your design.

ryanmiglavs commented:
I added 4pt to the width of each menu, hopefully that gives enough space for the full menu text. If not, let me know what version of Sketch you're using so I can test.

Also updated the folder name to 'Design'.

Cheers!

TwoLivesLeft commented:
I looked into the symbols of the old version of the file and it was just a bit small

<img width="296" alt="image" src="https://user-images.githubusercontent.com/816857/50810476-bdde7280-1359-11e9-870b-7f3b35c2334a.png">

Will test out the new version
